cmd/config.go

```
@@ -218,51 +238,69 @@ func cfgFilesAdd(dir string) (cfg *Config, err error) {
 			continue
 		}
 
-		if c.ChainID == "" && c.Key == "" && c.RPCAddr == "" {
-			p := &relayer.Path{}
-			if err = json.Unmarshal(byt, p); err != nil {
-				fmt.Printf("failed to unmarshal file %s, skipping...\n", pth)
-				continue
-			}
+		if err = cfg.AddChain(c); err != nil {
+			fmt.Printf("%s: %s\n", pth, err.Error())
+			continue
```

this should return the error

---

will update